examples/stress-nodes.js

```diff
@@ -0,0 +1,21 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { NodeDisruptor } from '../src/stress.js';
+
+export default function () {
+  const k8sClient = new Kubernetes()
+  const nodes = k8sClient.nodes.list().map(node => node.name)
+  const stress = new NodeDisruptor(k8sClient)
+  
+  // stress the nodes of the cluster
+  stress.stress(
```

Comment:
I should probably change the name of the `stress` variable to `nodeDisruptor` to prevent this ugly `stress.stress(...)`.